package.json

```diff
@@ -7,7 +7,9 @@
     "pretest": "yarn lint",
     "test": "lerna run test --stream",
     "prettier": "prettier --write **/*.{js,ts,tsx,json,css,scss,md,yml}",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "prepack": "cp README.md packages/cli/",
+    "postpack": "rm packages/cli/README.md"
```

Comment:
don't think cp and rm work properly on windows

Reply:
They're usually used throughout package.json scripts. Should be fine imo

Reply:
A lot of package.json scripts blow up on windows due to Unix shell commands. I'd recommend testing.

Reply:
Adding a `cp` script under windows 10 bails for me with:
```
'cp' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
`&&` works because npm uses cmd in the background afaik

we're already using `rimraf` so might as well use that here and then some non bloated copy cli (looking at `cpy-cli`)

Reply:
I'll push to this later